projects/testapp/CMakeLists.txt

```diff
@@ -1,27 +1,42 @@
 cmake_minimum_required(VERSION 3.14)
 project(testapp)
 
+set(SRC_DIR  ${CMAKE_CURRENT_LIST_DIR}/src)
+set(INCLUDE_DIR  ${CMAKE_CURRENT_LIST_DIR}/include)
```

Comment:
anti-best-practice. one more layer of abstraction that obfuscate whats really behind the variable